spec/syntax.md

```diff
@@ -216,17 +214,20 @@ respectively.
 
 Whitespace in _text_, including tabs, spaces, and newlines is significant and MUST
 be preserved during formatting.
-Embedding a _pattern_ in curly brackets ensures that _messages_ can be embedded into
+
+When a _pattern_ is quoted by embedding the _pattern_ in curly brackets, the
+resulting _message_ can be embedded into
 various formats regardless of the container's whitespace trimming rules.
+Otherwise, care must be taken to ensure that pattern-significant whitespace is preserved.
 
 > **Example**
 > In a Java `.properties` file, the values `hello` and `hello2` both contain
 > an identical _message_ which consists of a single _pattern_.
 > This _pattern_ consists of _text_ with exactly three spaces before and after the word "Hello":
 >
 > ```properties
-> hello = {   Hello   }
-> hello2={   Hello   }
+> hello = {{{{   Hello   }}}}
+> hello2=\   Hello  \ 
```

Comment:
LOL. I already had a hello2 I guess...

The space trimming here is an artifact of .properties, not of MF2. The problem is to get intentional spaces to the MF2 formatter. The `hello2` example shows how to do that in a properties file (a problem that already exists for users of this format--look for it on stackoverflow, for example).

From an MF2 perspective, `   Hello   ` is perfectly valid and clear about space handling. We don't need to solve every format's way of protecting the spaces in order for MF2 to be done. As you point out, the old syntax did guard against that... at the cost of making the message `Hello` invalid.

Reply:
This is a good point. I'll leave it unresolved so that I have a better chance of finding it again later. It SGTM. (And `.properties` is still weird (and still popular).)